### DISCUSSION TITLE
Are there any projects that use Observable-Hooks?

### DISCUSSION THREAD

**Discussion by pastelmind** [crimx/observable-hooks#111] [Q&A]
I came across Observable-Hooks while looking for ways to integrate RxJS with React. This library looks fantastic:+1:. I also checked out the example projects, but I'd like to see bigger projects using Observable-Hooks. Specifically, I want to know how applications handle errors in streams, as well as logging.

I did check out [ext-saladict](https://github.com/crimx/ext-saladict)--which is impressive, by the way. But I couldn't find any code that deals with errors occuring inside a stream.

Are there any projects that may be helpful for me?

**Comment by crimx** [comment]
Hi! ext-saladict uses it mainly for animations so error handling is not really required. Based on my observations on our non-open-sourced projects and some of the projects from issues I find people with RxJS background tend to use idiomatic RxJS way to handle errors(e.g. sub-stream with `catchError` operator). `useRenderThrow` is also useful if you want to handle errors in React ErrorBoundary.
